core/app/models/spree/app_configuration.rb

```diff
@@ -381,6 +381,16 @@ def tax_adjuster_class
       @tax_adjuster_class ||= Spree::Tax::OrderAdjuster
     end
 
+    # Allows providing your own classes for validate the checkout stard process
```

Comment:
_ stard_ looks like a typo?

also "for validate" might be better as "for validating"